src/csharp/Grpc.Tools/build/_protobuf/Google.Protobuf.Tools.targets

```diff
@@ -75,6 +75,10 @@
       <!-- Next try OS and CPU resolved by ProtoToolsPlatform. -->
       <Protobuf_ToolsOs Condition=" '$(Protobuf_ToolsOs)' == '' ">$(_Protobuf_ToolsOs)</Protobuf_ToolsOs>
       <Protobuf_ToolsCpu Condition=" '$(Protobuf_ToolsCpu)' == '' ">$(_Protobuf_ToolsCpu)</Protobuf_ToolsCpu>
+ 
+      <!-- Use x64 on macosx arm64 until a native protoc is shipped -->
+      <Protobuf_ToolsCpu Condition=" '$(Protobuf_ToolsCpu)' == 'arm64' and '$(Protobuf_ToolsOs)' == 'macosx' ">x64</Protobuf_ToolsCpu>
```

Comment:
If you want to override the cpu architecture, you should do it in the C# code
(e.g. in the https://github.com/grpc/grpc/blob/master/src/csharp/Grpc.Tools/ProtoToolsPlatform.cs), not here.

The msbuild files already way too complex, they weren't designed for hacks like this and they are also much harder to test than the C# code.